src/widgets/AfterInteractions.js

```diff
@@ -7,9 +7,14 @@ import { SUSSOL_ORANGE } from '../globalStyles/index';
 
 export const AfterInteractions = ({ children, placeholder }) => {
   const [ready, setReady] = useState(false);
+  const handler = useRef();
 
   useEffect(() => {
-    InteractionManager.runAfterInteractions(() => setReady(true));
+    handler.current = InteractionManager.runAfterInteractions(() => {
+      setReady(true);
+      handler.current = null;
+    });
+    return () => handler.current?.cancel();
```

Comment:
stop doing stuff if unmounted, basically

Reply:
looks fine - but on my tablet the edit sensor page just spins endlessly... reverting these changes makes it work

Reply:
🤷‍♂️  can't get it to either work or fail reliably. The fridge display is also spinning most of the time, so there's something amiss with the interactions, just not sure what.